frontend/lib/resolution/scope-queries.cpp

```diff
@@ -1692,7 +1759,8 @@ static bool lookupInScopeViz(Context* context,
                             context, scope, /* receiverScopes */ {},
                             resolving, name, config,
                             checkedScopes, result,
-                            idForWarnings);
+                            idForWarnings,
+                            /* allowCached */ false);
```

Comment:
Why is `allowCached` false here?

Reply:
Having the caching turned off here simplifies things by keeping the potential for query recursion for mutually recursive modules within a single query here. `publicSymbolsForModule` uses the queries that build `ResolvedVisibilityScope`, and the `false` here prevents the queries that build `ResolvedVisibilityScope` from using `publicSymbolsForModule`.

I'll add a comment about this.